packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -137,16 +129,21 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
 
   connectedCallback() {
     registerComponentForInit(this, this.standaloneEngineId);
-    this.addEventListener(
-      'suggestionlistrender',
-      this.handleSuggestionListEvent
-    );
+
+    this.addEventListener('inputvaluechange', this.handleInputValueChange);
+    this.addEventListener('submitsearch', this.handleSubmit);
+    this.addEventListener('showsuggestions', this.handleShowSuggestions);
```

Comment:
better no?, it doesn't need to start by `handle`
```suggestion
    this.addEventListener('showsuggestions', this.showSuggestions);
```

Reply:
sure, I thought I would start it by handle for consistency thats all